pool/test/filter_test.ml

```diff
@@ -534,3 +563,79 @@ let create_filter_template_with_template _ () =
   in
   Lwt.return_unit
 ;;
+
+let filter_with_admin_value _ () =
+  let%lwt () =
+    let open Utils.Lwt_result.Infix in
+    let%lwt experiment_id =
+      Experiment.find_all Test_utils.Data.database_label ()
+      ||> CCList.hd
+      ||> fun { Experiment.id; _ } -> id |> Experiment.Id.to_common
+    in
+    let%lwt contact = TestContacts.get_contact 0 in
+    let admin = Test_utils.Model.create_admin () in
+    let%lwt () =
+      CustomFieldData.(
+        answer_nr_of_siblings ~answer_value:3 [ contact ]
+        @ answer_nr_of_siblings ~answer_value:1 ~admin [ contact ])
+      |> Lwt_list.iter_s
+           (Pool_event.handle_event Test_utils.Data.database_label)
+    in
+    let find =
+      Filter.find_filtered_contacts Test_utils.Data.database_label experiment_id
+    in
+    let search filter =
+      filter
+      |> CCOption.pure
+      |> find
+      ||> CCResult.get_exn
+      ||> CCList.find_opt (Contact.equal contact)
+      ||> CCOption.is_some
+    in
+    let%lwt should_not_contain =
+      Filter.create None (nr_of_siblings ~nr:3 ()) |> search
+    in
+    let%lwt should_contain =
+      Filter.create None (nr_of_siblings ~nr:1 ()) |> search
+    in
+    let res = should_contain && not should_not_contain in
+    Alcotest.(check bool "succeeds" true res) |> Lwt.return
+  in
+  Lwt.return_unit
+;;
+
+let no_admin_values_shown_to_contacts _ () =
+  let%lwt () =
+    let open Utils.Lwt_result.Infix in
+    let open Custom_field in
+    let%lwt contact = TestContacts.get_contact 0 in
+    let%lwt custom_fields =
+      find_all_by_contact
+        Test_utils.Data.database_label
+        (Pool_context.Contact contact)
+        (Contact.id contact)
+      ||> fun (grouped, ungrouped) ->
+      ungrouped
+      @ CCList.flat_map Group.Public.(fun group -> group.fields) grouped
+    in
+    let res =
+      let open CCOption in
+      let open Answer in
+      custom_fields
+      |> CCList.filter (function
```

Comment:
If you expose the `admin_value` function of the Answer model, the "res" variable could be created with:
```ocaml
let res =
      let open CCOption in
      let open Answer in
      let open Public in
      custom_fields
      |> CCList.filter (function
           | Public.Boolean (_, answer) -> answer >>= admin_value |> is_some
           | MultiSelect (_, _, answer) -> answer >>= admin_value |> is_some
           | Number (_, answer) -> answer >>= admin_value |> is_some
           | Select (_, _, answer) -> answer >>= admin_value |> is_some
           | Text (_, answer) -> answer >>= admin_value |> is_some)
      |> CCList.is_empty
    in
```

Reply:
when `admin_value` is exposed e.g. the following line has to be updated as well:
https://github.com/uzh/pool/pull/87/files#diff-5670707531b84988327d7d674c67899d1aa64f400c94f44d8b85bc5a318845c8R143
(`Answer.` withing the curly brackets)
```
{ Answer.id = answer_id; value = Some 3; admin_value } |> CCOption.pure
```